Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Application/Modules/ApplicationModule.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Autofac;
+using MvvmCross.Core.ViewModels;
+using MvvmSeed.Application.ViewModels;
+
+namespace MvvmSeed.Application.Modules
+{
+    public class ApplicationModule : Module
```

Comment:
In the near future, if you are going to add another Module consider adding a description comment on each header (of what should be registered in that module). I've found it very useful in CS3 for example. Just an idea.